app/Http/Requests/PostRequest.php

```diff
@@ -30,7 +30,7 @@ public function rules()
             'why_participated' => 'nullable|string',
             'text' => 'required|nullable|string|max:256',
             'quantity' => 'nullable|integer',
-            'file' => 'image|dimensions:min_width=400,min_height=400',
+            // 'file' => 'image|dimensions:min_width=400,min_height=400',
             'status' => 'in:pending,accepted,rejected',
```

Comment:
Do we need this validation? With this validation, I wasn't able to upload a photo in Rogue Admin because it threw a `422 (Unprocessable Entity)` error.

Reply:
i'm fine with removing this. but can we add a `TODO` to figure out the file validation?

Reply:
Wait, we definitely want this! (Unless it's triggering for `text` posts? What's the error?)

Reply:
@DFurnes when I try to upload a photo in Rogue admin via the uploader, with this validation, it throws a `422 (Unprocessable Entity)` :-/

Reply:
What are the details of the validation error?

Reply:
also can we remove the requirement for `text` since not all posts will have this.

Reply:
oh right @sbsmith86 ! 

@DFurnes it doesn't give much more info. than that :-/ 
![screen shot 2018-04-05 at 2 08 32 pm](https://user-images.githubusercontent.com/9019452/38383570-dc6b8eae-38da-11e8-958f-a2d249b4e8fa.png)